dvci/shc-web-verifier
# Download clicky data

# Summary
Creates a GitHub workflow that runs a Python script each Sunday and the first of each month. The script gathers the latest visitor data from Clicky and adds it to the existing visitor data in the repository. The script then creates a JSON file of the updated visitor data (`clickyData.json`) and a JPEG of a Matplotlib representation of the data (`clickyGraph.jpg`).

# Testing Guidance
* If you have not done so already, follow the [MITRE Python SSL Setup](https://awesome.pages.mitre.org/full-stack-mitre/dev-environment/programming-languages/python/) to install the MITRE certs into your Python environment.
* Make sure you have the necessary imports for running the script - `requests` and `matplotlib` (you may already have them installed)
* Run the `runClickyScript.py` file within the Python shell and check that the script runs successfully. Then, check that the files `clickyData.json` and `clickyGraph.jpg` are created. The `__pycache__` directory will also get created, but it should be git ignored.
